apps/dh/app-dh/project.json

```diff
@@ -30,6 +30,11 @@
             "input": "libs/dh/globalization/assets-localization/src/assets",
             "output": "assets"
           },
+          {
+            "glob": "**/*",
+            "input": "libs/dh/core/assets-core",
+            "output": "assets"
+          },
```

Comment:
suggestion: Let's use `dh-shared-assets` library for the logos. As static files that don't change often, I don't see the need to scatter them between in multiple libraries.